From 8354090fcd573be361b7bacaeed19801fbe24a20 Mon Sep 17 00:00:00 2001
From: Francois Choleau <francois.choleau@st.com>
Date: Fri, 14 Jun 2024 03:45:49 -0400
Subject: [PATCH 0908/1695] dts: stm32: update stm32mp235-dk.dts

Update as per stm32mp257f-dk

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f6362f876924fa025e88ed7cd59fbfb7d83b0ab4 dts: stm32: update stm32mp235-dk.dts]
Signed-off-by: Francois Choleau <francois.choleau@st.com>
Change-Id: I6347f2a3d7209c192bb9178a125af54ba98674e9
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/388622
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Tested-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp235f-dk.dts | 14 +++++++++++++-
 1 file changed, 13 insertions(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp235f-dk.dts b/core/arch/arm/dts/stm32mp235f-dk.dts
index dec4888d3..010469b7f 100644
--- a/core/arch/arm/dts/stm32mp235f-dk.dts
+++ b/core/arch/arm/dts/stm32mp235f-dk.dts
@@ -37,7 +37,7 @@
 
 	memory@80000000 {
 		device_type = "memory";
-		reg = <0x0 0x80000000 0x1 0x00000000>;
+		reg = <0x0 0x80000000 0x0 0x80000000>;
 	};
 
 	shadow-prov {
@@ -53,6 +53,10 @@
 	cpu-supply = <&vddcpu>;
 };
 
+&hash {
+	status = "okay";
+};
+
 &hsi_calibration {
 	status = "okay";
 };
@@ -324,6 +328,10 @@
 	status = "okay";
 };
 
+&pka {
+	status = "okay";
+};
+
 &pwr {
 	status = "okay";
 	wakeup-gpios = <&gpioa 0 GPIO_ACTIVE_HIGH>, <0>, <0>, <0>, <0>, <0>;
@@ -405,6 +413,10 @@
 	status = "okay";
 };
 
+&saes {
+	status = "okay";
+};
+
 &scmi_regu {
 	voltd-vddcore {
 		reg = <VOLTD_SCMI_STPMIC2_BUCK2>;
-- 
2.47.2

